astronomer/providers/amazon/aws/operators/batch.py

```diff
@@ -27,12 +27,13 @@ class BatchOperatorAsync(BatchOperator):
     :param job_name: the name for the job that will run on AWS Batch (templated)
     :param job_definition: the job definition name on AWS Batch
     :param job_queue: the queue name on AWS Batch
-    :param overrides: the `containerOverrides` parameter for boto3 (templated)
+    :param overrides: DEPRECATED and removed in 8.0.0, use container_overrides instead with the same value.
```

Comment:
`DEPRECATED and removed in 8.0.0 ` might be confusing for the reader here because the user may or may not know that 8.0.0 we are referring for oss version